### ISSUE TITLE
app-crypt/asleap-2.2_p20201128 does not respect CFLAGS

### ISSUE THREAD

**Issue by asarubbo** [pentoo/pentoo-overlay#759]
https://blogs.gentoo.org/ago/2020/07/04/gentoo-tinderbox/

Issue: app-crypt/asleap-2.2_p20201128 does not respect CFLAGS.
Discovered on: amd64 (internal ref: pentoo_tinderbox)

NOTE:
To reproduce this issue you may want to add "-frecord-gcc-switches" to CFLAGS/CXXFLAGS/FFLAGS/FCFLAGS. If this is not something about c/c++ context see this bug as an hint to hide the QA warning (with QA_FLAGS_IGNORED) where is not possible to respect CFLAGS.

At the time of writing, github-cli does not support attachments. I'm using gists but they can be deleted at a certain point. If you want to save them, please create attachments on your own.

build log: https://gist.github.com/22436606882ce882e16f545c73ccbd46

**Comment by blshkv** [comment]
Yes, yes, we know. Our ebuilds are not perfect, that's why it is still not in Gentoo. Please create a pull request with a fix you care

**Comment by blshkv** [comment]
seems fixed
